examples/Console/TestOTelShimWithConsoleExporter.cs

```diff
@@ -36,7 +36,10 @@ internal static object Run(OpenTelemetryShimOptions options)
 
             var tracer = TracerProvider.Default.GetTracer("MyCompany.MyProduct.MyWebServer");
             var span = tracer.StartSpan("parent span");
-            span.SetAttribute("my", "value");
+            span.SetAttribute("mystring", "value");
+            span.SetAttribute("myint", 100);
```

Comment:
Yeah!!!

Reply:
I second that.

Reply:
It's definitely a 🎉, but stored as an object underneath so there's still a boxing penalty. It could have been a 🚀 🤣